package.json

```diff
@@ -106,7 +106,8 @@
     "vue": "^2.5.17",
     "vue-server-renderer": "^2.5.17",
     "web-vitals": "^0.2.4",
-    "webpack": "5.76.0",
+    "webpack": "5.94.0",
     "when": "^3.7.8"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Ah, `corepack` added it automatically. :laughing: 
Is it fine, or should I remove it?

Reply:
Let's keep it. Eventually we'll want to move off yarn 1 but its good to document which version was used.